app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb

```diff
@@ -12,14 +12,33 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Credential < Mana
 
   FRIENDLY_NAME = "Ansible Automation Inside Credential".freeze
 
-  def self.provider_params(params)
-    super.merge(:organization => ManageIQ::Providers::EmbeddedAnsible::AutomationManager.first.provider.default_organization)
+  include ManageIQ::Providers::EmbeddedAnsible::CrudCommon
+
+  def self.params_to_attributes(_params)
+    raise NotImplementedError, "must be implemented in a subclass"
   end
 
   def self.notify_on_provider_interaction?
     true
   end
 
+  def self.raw_create_in_provider(_manager, params)
+    create!(params_to_attributes(params))
+  end
+
+  def self.notification_excludes
+    [:password, :auth_key, :service_account]
```

Comment:
This list is already enumerated elsewhere via `encrypted_columns` class method...it's probably preferable to just put that list right in the base class, and then you don't need these overrides at all

Reply:
@Fryguy I think what you are suggesting here and pushing this up to `CrudCommon` will work, but I don't think this will work in the specialized classes, since a lot of them make use of `alias_attribute`, which `PasswordMixin` does not support.